src/inference/include/openvino/runtime/intel_gpu/properties.hpp

```diff
@@ -63,6 +63,14 @@ static constexpr Property<std::map<std::string, uint64_t>, PropertyMutability::R
  */
 static constexpr Property<bool> enable_loop_unrolling{"GPU_ENABLE_LOOP_UNROLLING"};
 
+/**
+ * @brief Turning on this key disables winograd convolution in iGPU.
```

Comment:
No need to mention iGPU here - it's implementation detail.

Reply:
Applied